README.md

```diff
@@ -955,6 +955,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [gonic](https://github.com/sentriz/gonic) - Lightweight music streaming server. Subsonic compatible. `GPL-3.0` `Go`
 - [Groove Basin](https://github.com/andrewrk/groovebasin) - Music player server with a web-based user interface inspired by Amarok 1.4. `MIT` `Nodejs`
 - [koel](http://koel.phanan.net/) - Personal music streaming server that works. ([Source Code](https://github.com/phanan/koel)) `MIT` `PHP`
+- [KooZic](https://koozic.net/) - Music server with powerful playlist features. Subsonic compatible ([Demo](https://demo.koozic.net/public), [Source Code](https://github.com/DocMarty84/koozic)) `LGPL` `MIT` `Python`
```

Comment:
I didn't see a reference to the MIT license in https://github.com/DocMarty84/koozic/blob/v3/LICENSE in which case change
```
 `LGPL` `MIT` `Python` 
```
to
```
`LGPL-3.0` `Python`
```
If MIT does need to be included it should be 
```
`LGPL-3.0/MIT` `Python`
```